Documentation/PageTsconfig/Mod/WebInfo.rst

```diff
@@ -74,41 +74,24 @@ Example: Override the field definitions in the info module
 menu.function
 =============
 
-..  confval:: menu.function
-    :name: mod-web-info-menu-function
-    :type: array
-
-    Disable elements of the "Function selector" in the document header of the module. The keys for single
-    items can be found by browsing *System > Configuration > $GLOBALS['TBE_MODULES_EXT']*.
+..  versionchanged:: 12.0
+    The TSconfig option :tsconfig:`mod.web_info.menu.function` has been removed
+    with TYPO3 v12.0.
 
-    ..  figure:: /Images/ManualScreenshots/Info/FunctionMenuInfoModule.png
-        :alt: The function menu of the Info module
+..  _pageblindingfunctionmenuoptions-webinfo-migration:
 
-        The function menu of the Info module
+Migration from menu.function to options.hideModules
+---------------------------------------------------
 
-    ..  warning::
+..  todo: link to options.hideModules once it is documented
 
-        Blinding the function menu items is not hardcore access control! All it
-        does is hide the possibility of accessing that module functionality
-        from the interface. It might be possible for users to hack their way
-        around it and access the functionality anyways. You should use the
-        option of blinding elements mostly to remove otherwise distracting options.
+..  code-block:: typoscript
 
-..  _pageblindingfunctionmenuoptions-webinfo-example:
+    # before
+    mod.web_info.menu.function.TYPO3\CMS\Info\Controller\TranslationStatusController = 0
 
-Example: Remove some options from the functions menu
-----------------------------------------------------
+    # after
+    options.hideModules := addToList(web_info_translations)
 
-..  code-block:: typoscript
-    :caption: EXT:site_package/Configuration/page.tsconfig
-
-    mod.web_info.menu.function {
-        # Disable item "Log"
-        TYPO3\CMS\Belog\Module\BackendLogModuleBootstrap = 0
-        # Disable item "Pagetree Overview"
-        TYPO3\CMS\Info\Controller\PageInformationController = 0
-        # Disable item "Localization Overview"
-        TYPO3\CMS\Info\Controller\TranslationStatusController = 0
-        # Disable item "Linkvalidator"
-        TYPO3\CMS\Linkvalidator\Report\LinkValidatorReport = 0
-    }
+You can find the names of all TypoScript modules in
+:file:`EXT:info/Configuration/Backend/Modules.php`.
```

Comment:
Please use t3src for easy access. And I am not sure, that TypoScript modules are registered in EXT:info.

Reply:
We are in the web_info module here and it is registered there, looked it up

Reply:
So we have two options.hideModules, one in page TSconfig and one in user TSconfig?